assets/ambientcg.com/SkyOnlyHDRI023_2K-TONEMAPPED.jpg.import

```diff
@@ -18,9 +18,9 @@ dest_files=["res://.godot/imported/SkyOnlyHDRI023_2K-TONEMAPPED.jpg-bcb5464e5ed4
 [params]
 
 compress/mode=2
+compress/high_quality=false
```

Comment:
I wonder what happens if we set this to true, seeing quality of the panoramic sky seems so-so

Also double check if mipmaps are off

Reply:
Set to high-quality and disabled mipmap generation.